resources/views/blog/index.blade.php

```diff
@@ -0,0 +1,68 @@
+<?php ?>
+@extends('layouts.app')
+@section('content')
+
+<button type="button" class="btn btn-primary" data-toggle="modal" data-target="#createModal">Create Blog</button>
+
+<div class="modal fade" id="createModal" tabindex="-1" aria-labelledby="createModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="createModalLabel">New blog</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <form action="{{route('blog.store')}}" name="form-create" method="POST" enctype="multipart/form-data">
+        @csrf
+        <div class="modal-body">
+          <div class="form-group">
+              <label for="blog-title" class="col-form-label">Title:</label>
+              <input type="text" class="form-control" id="blog-title" name="title">
+            </div>
```

Comment:
Please fix indent text and apply for same line

Reply:
I fixed, please help me review this task